specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/examples/DistributedAvailabilityGroupsSetRole.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "managedInstanceName": "testcl",
+    "distributedAvailabilityGroupName": "dag",
+    "api-version": "2022-05-01-preview",
+    "parameters": {
+      "instanceRole": "Primary",
+      "roleChangeType": "Forced"
```

Comment:
if we just update or add this example file, in the dSMainDev PR, can you just add this example file in the example folder? 
for other example files, please refer to previous versions' example folder.

Reply:
Sure, I'll remove duplicates and point to the old examples